src/Biconomy.js

```diff
@@ -250,6 +257,20 @@ function Biconomy(provider, options) {
   }
 }
 
+Biconomy.prototype.getSignerByAddress = function(userAddress) {
```

Comment:
Where will this be used?

Reply:
This will be used to pass as a signer object when you are instantiating a Contract object. A signer or provider is needed, so we'll just provide this signer with getAddress method implementation so ethers don't throw error.